patches/tock/01-persistent-storage.patch

```diff
@@ -42,7 +42,7 @@ index fe493727..105f7120 100644
  
      platform.pconsole.start();
 diff --git a/chips/nrf52/src/nvmc.rs b/chips/nrf52/src/nvmc.rs
-index 5abd2d84..5a726fdb 100644
+index 60fc2da8..77e7423d 100644
```

Comment:
Is there a plan to upstream (part of) this patch to mainline Tock? Or is it too much WIP?

Reply:
The initial plan was https://github.com/tock/tock/pull/1457 now obsoleted in favor of https://github.com/tock/tock/pull/1467 but currently blocked by design considerations regarding chips implementing syscalls. There's also some chance for this to progress given https://groups.google.com/g/tock-dev/c/aduN7fHWXdI/m/Lk3bmMVxBQAJ. The first step to provide applications access to a persistent storage is to check that access is permitted. Maybe once this first step is resolved, progress will unblock on the second step.